pkg/bootstrapper/bootstrapper.go

```diff
@@ -403,7 +406,10 @@ func (wmcb *winNodeBootstrapper) parseIgnitionFileContents(ignitionFileContents
 	}
 
 	// Generate the full list of kubelet arguments from the arguments present in the ignition file
-	wmcb.kubeletArgs = wmcb.generateInitialKubeletArgs(args)
+	wmcb.kubeletArgs, err = wmcb.generateInitialKubeletArgs(args)
+	if err != nil {
+		return fmt.Errorf("cannot generate initial kubelet args: %w\n", err)
```

Comment:
It's not usual to add a new line to an error string like this, what's the intention here?

Also, is there a particular purpose to wrapping the error? Are you likely to unwrap later? If not, just use `%v` instead, it's more idiomatic